src/mixins/toggleStatusbarColor.js

```diff
@@ -13,14 +14,18 @@ export default {
     for (const selector of metaSelectors) {
       document.querySelector(selector).content = this.newStatusbarColor
     }
-    StatusBar.setStyle({ style: StatusBarStyle.Light })
+    StatusBar.setStyle({
+      style: Vue.prototype.$isIOS() ? StatusBarStyle.Dark : StatusBarStyle.Light,
```

Comment:
you can change to `this.$isIOS` since this is a mixin and remove the Vue import at the top